### DISCUSSION TITLE
Modify the schedule HTML markup

### DISCUSSION THREAD

**Discussion by acanza** [nhn/tui.calendar#1034] [Q&A] [answered]
Hi! First of all, thank you for this awesome library 👏🏻

I'm looking for a way to replace the default schedule markup with my own to get something as I show below, but I don't find any method or any options in the calendar API 😔
<img width="1254" alt="Captura de pantalla 2022-03-04 a las 9 09 05" src="https://user-images.githubusercontent.com/3978410/156728050-c7464eec-dd73-480c-a4e9-ea78da424195.png">

Thanks in advance!

**Comment by TheClassYoga** [comment]
Yes please!!! Looking forward to it!

**Comment by acanza** [comment, marked as answer]
Hi again! Finally, I got it! 😊

I replaced the markup through "schedule" property in the "template" calendar option.
